src/gui/mzroll/database.cpp

```diff
@@ -84,36 +84,86 @@ void Database::loadCategories() {
 	return;
 }
 
-bool Database::addCompound(Compound* c) {
-    if(c == NULL) return false;
+bool Database::addCompound(Compound* newCompound)
+{
+    if(newCompound == nullptr)
+        return false;
+
     bool compoundAdded = false;
 
-    //new compound id .. insert into compound list
-    if (!compoundIdMap.count(c->id)) {
-        compoundIdMap[c->id] = c;
-        compoundsDB.push_back(c);
+    // lambda that checks if a compound contains PRM information
+    auto isPRM = [](const Compound* cpd) -> bool {
```

Comment:
A compound cannot be 'PRM'. Could you replace `isPRM` with `hasSpectralInfo` or `hasFragmentationInfo` to make it more explicit? You can suggest other names as well. These are just off the top of my head

Reply:
I agree. This should have been renamed to something more apt. But since then I've removed this piece of code entirely (we agreed to not check for any categories and simply append a counter after a compound's name). You may want to refresh your page.

Reply:
I'm going commit-by-commit. Forgot about that change, sorry. I'll remove the comment